taxcalc/parameters.py

```diff
@@ -689,11 +689,11 @@ def num_years(self):
 
     @property
     def parameter_warnings(self):
-        return self.errors or ""
+        return self.errors or {}
```

Comment:
The parameter errors are a dictionary that looks like this:

```python
{'AMEDT_rt': ['AMEDT_rt[year=2021] 1.8 > max 1 ']}
```

This change sets the default value to be a dictionary to keep the type consistent.